tests/__snapshots__/cdn.test.ts.snap

```diff
@@ -758,7 +757,6 @@ video {
   --surface-lg: #ffffff;
   --surface-md: #ffffff;
   --surface-xl: #ffffff;
-  --surface-dash-widget: #ffffff;
```

Comment:
Due to the removal of a style here, I think we may want a `major` version update, as to not surprise consumers.  Even if the only consumers are us, internally, it'd be nice to follow semver.  What are your thoughts?

Reply:
That seems like a good call to me! I've updated the PR to make it a major version bump instead then.